### ISSUE TITLE
package notes

### ISSUE THREAD

**Issue by waclaw66** [cyqsimon/ntfysh-spec#2]
- /docs/ endpoint doesn't work, it shows just a blank page, ntfy_2.1.2_linux_amd64.rpm package from the author provides it correctly
- it would be better to use the same package name `ntfy` instead of `ntfysh` to be compatible with author's packages

**Comment by cyqsimon** [comment]
> /docs/ endpoint doesn't work, it shows just a blank page, ntfy_2.1.2_linux_amd64.rpm package from the author provides it correctly

I think it's the same problem as #1, although I remember not wanting to enable it due to some dependency issue. I will try and get back to you.

> it would be better to use the same package name `ntfy` instead of `ntfysh` to be compatible with author's packages

I swear I remember the `ntfy` name being taken up by some python package on RHEL... But rechecking now it seems like that was never the case, which is ***very*** strange because I'm 25 and my memory usually doesn't fail me like this. If there was never a conflict, I do agree that the simple `ntfy` name is better.

That being said, it's nothing unusual for the packaging name to be different to the project name. So given that changing the package name now will cause existing users to stop receiving updates, I'm inclined to keep it as is in lieu of a very good reason otherwise.

**Comment by cyqsimon** [comment]
So yes, my recollection was correct. There are dependency issues, and very annoying ones too.

So on all Fedora and EL9 chroots, Python>=3.8 is shipped by default, so there is no issue.

On EL7, Python 3.8 is available as `rh-python38` from CentOS SCLo RH repo. But for some reason it's failing to install in the COPR chroot despite the repo being available. I'll have to do some digging.

On EL8, Python3.9 is available, but by default Python 3.6 is also installed, which is the version `python3` points to. On your own machine you would use `alternatives` to make it point to `python3.9` instead, but in COPR chroot you don't have sufficient permission... There is provision for this situation in ntfy's `Makefile`, but it is hard-coded for 3.8, so I will need to submit a patch for that.

The TLDR is, this is not getting fixed this version. Sorry ¯\\\_(ツ)\_/¯. I'll try to get everything ready by next release.

**Comment by waclaw66** [comment]
Thanks for considering the changes, looking forward for the next version.

**Comment by cyqsimon** [comment]
The patch has been submitted. https://github.com/binwiederhier/ntfy/pull/665

**Comment by cyqsimon** [comment]
Specifying a python binary override is finally possible after binwiederhier/ntfy#949. I will try to fix this next release.

**Comment by cyqsimon** [comment]
Should be fixed in [2.8.0-2](https://copr.fedorainfracloud.org/coprs/cyqsimon/ntfysh/build/6669096/).

See also: e585474, ac5c1cb.
